src/components/portrait/portrait-initials.d.ts

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
```

Comment:
This file is not needed

Reply:
https://github.com/Sage/carbon/pull/3816/files#r609671352